edgexfoundry/app-functions-sdk-go
# feat: Added support for custom app service to have custom service metrics

closes #1084

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

     https://github.com/edgexfoundry/edgex-docs/pull/739

## Testing Instructions
Run EdgeX non-secure stack with Device Virtual
Build the app template
Run the app template (./new-app-service)
Verify the following logs every 30secs
```
level=DEBUG ts=2022-04-20T00:40:53.1809883Z app=new-app-service source=reporter.go:165 msg="Publish 1 metrics to the 'edgex/telemetry/new-app-service
' base topic"
level=DEBUG ts=2022-04-20T00:40:53.1810529Z app=new-app-service source=manager.go:117 msg="Reported metrics..."
```
## New Dependency Instructions (If applicable)
